src/libmongoc/src/mongoc/mongoc-crypt.c

```diff
@@ -681,71 +682,11 @@ _mongoc_crypt_new (const bson_t *kms_providers,
    mongocrypt_setopt_log_handler (
       crypt->handle, _log_callback, NULL /* context */);
 
-   /* Take options from the kms_providers map. */
```

Comment:
libmongocrypt's new API for configuring KMS providers: `mongocrypt_setopt_kms_providers` no longer requires the calling driver to parse out options. We can pass the KMS providers BSON document directly.